src/benchmark.cc

```diff
@@ -781,7 +798,14 @@ void RunBenchmark(const benchmark::internal::Benchmark::Instance& b,
         report.cpu_accumulated_time = cpu_accumulated_time;
         report.bytes_per_second = bytes_per_second;
         report.items_per_second = items_per_second;
+        report.arg1 = b.arg1;
```

Comment:
Hello dominic. I have already added the field SetComplexityN. If there is something else please let me know.